### DISCUSSION TITLE
Vercel color theme is unfriendly for astigmatism eye

### DISCUSSION THREAD

**Discussion by keidarcy** [vercel/community#220] [Help] [answered]
The high contrast theme maybe look good for most people. like this page https://vercel.com/support
But for astigmatism, it's suffering...
Could you please at least add a dark/light mode switcher... or dimmed color theme 🙏

**Comment by leerob** [comment, marked as answer]
Hey, thank you for this feedback. There is a theme switcher in the footer in the meantime 👍 
![CleanShot 2022-01-23 at 20 49 02@2x](https://user-images.githubusercontent.com/9113740/150713994-db3abc6d-0533-4c64-9253-108a6ecdd91a.png)

**Reply by keidarcy** [reply]
OH! Thanks!
